app/controllers/trainees/degrees/type_controller.rb

```diff
@@ -18,6 +19,10 @@ def create
 
     private
 
+      def authorize_trainee
+        authorize([:degrees, trainee])
+      end
```

Comment:
```suggestion
      def authorize_degree
        authorize(@degree)
      end
```

Reply:
For this to work the name has to be `authorize_trainee` as I override [base_controller.rb#L15](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/controllers/trainees/base_controller.rb#L15) from the `Trainees::BaseController`